spec/ravelin/authentication_mechanism_spec.rb

```diff
@@ -1,22 +1,44 @@
 require 'spec_helper'
 
 describe Ravelin::AuthenticationMechanism do
-  subject do
-    described_class.new(
-      {
-        password: {
-          password: "lol",
-          success: true
+  context 'for password authentication mechanisms' do
+    subject do
+      described_class.new(
+        {
+          password: {
+            password: "lol",
```

Comment:
In a public repo, should we be more professional than `lol`? I suggest `example`.

Reply:
I can take the opportunity to change the password, I'll put `super-secret-password` 😄